README.md

```diff
@@ -17,138 +19,143 @@ Before getting started with this project, please get yourself familiar with the
+```
+and for `adu_release` like this: 
+```shell
+adu_release=main
+```
+and for `project_root` like this:
+```shell
+project_root=~/
+```
 
-- Clone the Yocto (Poky) project
-    
-    ```shell
-    yocto_release=honister
-    adu_release=main
-    
-    # Clone project with Yocto configuration files
-    git clone <github url> -b <branchname> $proj_root
-
-    cd $project_root/yocto
+The first step for building the project is cloning this repository onto your device using the following command:
 
-    # Clone the Yocto Project (poky) into 'yocto' dir
-    git clone --depth 1 --branch $yocto_release git://git.yoctoproject.org/poky
-    ```
+1. Clone this repository onto your device:
+    
+```shell
+git clone https://github.com/Azure/iot-hub-device-update-yocto -b <branchname> $proj_root
+```
+2. Once you've cloned the project you next need to change into our "working directory" where the individual layers (in Yocto these layers build up to an image like a cake... or foundation.. or cake). 
+```shell
+cd $proj_root/iot-hub-device-update/yocto 
+```
+3. Once you're in the `yocto` directory we need to check out the Yocto Build "engine" or base layer so we can build with it. 
+```shell
+git clone --depth 1 --branch $yocto_release git://git.yoctoproject.org/poky
+```
+4. Next we need to checkout the rest of the dependency layers into the `yocto` directory
 
-- Clone SWUpdate meta layer. SWUpdate provides an image-based update that support dual-partition.
-  
+    1. Clone the SwUpdate meta layer 
     ```shell
-    # Clone swupdate meta layer 
     git clone --depth 1 --branch $yocto_release  https://github.com/sbabic/meta-swupdate
     ```
 
-- Clone the Open Embedded meta layer. This layer include many modules (or layers) needed for building a Linux-base system.
-
+    2. Clone the Open Embedded meta layer. This layer include many modules (or layers) needed for building a Linux-base system.
     ```shell
     git clone --depth 1 --branch $yocto_release  git://git.openembedded.org/meta-openembedded
     ```
 
-- Clone the Raspberry Pi meta layer. Since, the reference image that we are building is for a Raspberry Pi 3B+ hardware.
-
+    3. Clone the Raspberry Pi meta layer. Since, the reference image that we are building is for a Raspberry Pi 4 hardware.
     ```shell
     git clone --depth 1 --branch $yocto_release git://git.yoctoproject.org/meta-raspberrypi
     ```
 
-- Clone the Azure Device Update meta layer.
+5. Within the same directory we are now going to include the Device Update for IotHub layers which builds Device Update and provides those artifacts for the `meta-raspberrypi-adu` layer which integrates Device Update and modifies the image build instructions within `meta-raspberrypi` to output the `adu-base-image-<machine-name>.wic.gz` and `adu-update-image-<machine-name>.swu` which can be used to test out Device Update for IotHub. For more information on these layers and their outputs please read the `README.md` in each of the repos. 
```

Comment:
I'm trying not to wordsmith, but I think this is enough of a run-on sentence that it's actually quite difficult to follow. Can we break it up something like:

_5. Within the same directory we are now going to include the Device Update for IotHub layers which builds Device Update and provides those artifacts for the `meta-raspberrypi-adu` layer. The `meta-raspberrypi-adu` layer in turn integrates Device Update and modifies the image build instructions within `meta-raspberrypi` to output the `adu-base-image-<machine-name>.wic.gz` and `adu-update-image-<machine-name>.swu`. The outputted files can then be used to test out Device Update for IotHub._

Reply:
Somewhere my third-year english teacher is nodding her head and pulling out a red pen. I have a problem with run ons. There's likely more than one in the document. I'll go through and break them up into more logical components.